.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Consider adding `node_modules` directory to `.gitignore` file as it can be added by accident while setting up linters.